content/INTRODUCTION.md

```diff
@@ -0,0 +1,18 @@
+**Requirements**: No programming experience, at least 12 years old
+
+**Duration**: 
+
+**Standards**: K-12 Computer Science Framework Core Concepts and Practices
+* Algorithms and Programming
+  - Collaborating Around Computing
+  - Recognizing and Defining Computational Problems
+  - Developing and Using Abstractions
+  - Creating Computational Artifacts
+  - Testing and Refining Computational Artifacts
+
+## Web Development Curriculum
+
+Learn to code and start building professional, career-boosting websites and web applications.
+
+The `Web Development Curriculum` is designed to develop the logical, analytical, programming, and web development skills of individuals who want to become full-stack web developers. In this curriculum, learners will learn comprehensive concepts and principles on how to create dynamic websites and web applications using several tools and technologies such as `HTML`, `CSS`, `JavaScript`, `DOM`, `PHP`, and `PostgreSQL`. They will also experience solving and working on real-world projects and problems encountered by professional web developers.
```

Comment:
Is it Web Development Curriculum? I thought this is just a Web Dev course?

Reply:
@TenshiLyle This is an Introduction for the Curriculum Page that we want the learners to view first before the list of courses. Please let me know your thoughts about this. Thanks.